addon/background.js

```diff
@@ -113,14 +113,24 @@ browser.webRequest.onBeforeSendHeaders.addListener(function (info) {
 // Remove X-Frame-Options to allow any page to be embedded in an iframe
 chrome.webRequest.onHeadersReceived.addListener(function (info) {
   let headers = info.responseHeaders;
+  let madeChanges = false;
   for (let i = 0; i < headers.length; i++) {
     let name = headers[i].name.toLowerCase();
     if (name === "x-frame-options" || name === "frame-options") {
       headers.splice(i, 1);
-      return {"responseHeaders": headers};
+      i--;
+      madeChanges = true;
+    }
+    if (name === "content-security-policy") {
+      headers[i].value = headers[i].value.replace(/frame-ancestors[^;]*/i, "");
+      madeChanges = true;
     }
   }
-  return {};
+  if (madeChanges) {
+    return {"responseHeaders": headers};
+  } else {
+    return {};
```

Comment:
From the Circle-CI logs:

```
/home/circleci/project/addon/background.js
  131:10  error    Unnecessary 'else' after 'return'  no-else-return
```

I think it's saying we can just simplify to the following:

```js
if (madeChanges) {
  return {"responseHeaders": headers};
}
return {};
```